app/scripts/services/logLinks.js

```diff
@@ -65,15 +65,15 @@ angular.module('openshiftConsole')
         ")",
         "&_a=(",
           //"columns:!(_source),",
-          "columns:!(kubernetes_container_name,message),",
-          "index:'<%= namespace %>.<%= namespaceUid %>.*',",
+          "columns:!(kubernetes<%= use_cdm ? '.' : '_' %>container_name,message),",
```

Comment:
Trying to decide if we should `use_cmd:true` or just update the template to `kubernetes.container_name` and keep it simpler.  Per the discussion, it sounds like this is a certain update.  Less noise in this crazy string template may be nice.